sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/IndexingPolicy.java

```diff
@@ -86,6 +86,26 @@ public IndexingPolicy() {
         this.jsonSerializable = new JsonSerializable(objectNode);
     }
 
+    /**
+     * Gets whether the policy defined in the portal should be overwritten.
+     *
+     * @return the overwritePolicy
+     */
+    public Boolean isOverwritePolicy() {
```

Comment:
These APIs shouldn't be public, as native Java SDK user doesn't need to worry about whether to overwrite Indexing policy or not.

Reply:
If you need to access them in spring-data SDK, you can use ImplementationBridgeHelpers. There are multiple examples in other model classes on how to use them. You can check any of the Request Options classes.

Reply:
Done.